packages/inputs/src/Table.js

```diff
@@ -275,6 +275,13 @@ export class Table extends Input {
     }
   }
 
+  reset() {
+    this.currentRow = -1;
+    if (isSelection(this.selection)) {
+      this.selection.reset();
```

Comment:
The semantics of `reset` should be clarified. I think of the resetting the clause source as separate from a reset of a selection. The reset of the clause source should reset any interface state (like the brush extents, or text query string) but should not affect the selection state. The selection resolver instead is responsible for the selection state, and calls reset on the clause sources.

Reply:
Should be fixed now! I'm a bit unsure if what i'm doing for the `PanZoom.js` reset function is correct but from what I understand everything should be good in its current state!